autotest/utils/restful_return_check.py

```diff
@@ -15,28 +18,54 @@ def assert_chat_completions_batch_return(output, model_name):
         assert message.get('index') == 0
         assert len(message.get('message').get('content')) > 0
         assert message.get('message').get('role') == 'assistant'
+        if check_logprobs:
+            print(message.get('logprobs'))
+            len(message.get('logprobs').get('content')) == output.get(
+                'usage').get('completion_tokens')
+            for logprob in message.get('logprobs').get('content'):
+                assert_logprobs(logprob, logprobs_num)
+
+
+def assert_logprobs(logprobs, logprobs_num):
+    assert_logprob_element(logprobs)
+    assert len(logprobs.get('top_logprobs')) > 0 and type(
```

Comment:
type(logprobs.get(''top_logprobs')) == list should be checked first.
I think we should check `len(logprobs.get('top_logprobs')) == logprobs_num` rather than 
`len(logprobs.get('top_logprobs')) > 0 and len(logprobs.get('top_logprobs')) <= logprobs_num`

shouldn't we? @irexyc

Reply:
Yes, but currently the return num is incorrect. will add testcase after https://github.com/InternLM/lmdeploy/issues/1548